pvanalytics/features/clipping.py

```diff
@@ -79,6 +79,9 @@ def levels(ac_power, window=4, fraction_in_window=0.75,
     for more information.
 
     """
+    power = ac_power
```

Comment:
Only editorial, but I think it would be more clear as
```
power = ac_power.copy()
```
operate on `power`, then `reindex_like(ac_power)`

Reply:
I do like that better.